app/src/main/java/com/keylesspalace/tusky/adapter/StatusDetailedViewHolder.java

```diff
@@ -37,17 +39,73 @@ public StatusDetailedViewHolder(View view) {
     }
 
     @Override
-    protected void setCreatedAt(Date createdAt, Date editedAt, StatusDisplayOptions statusDisplayOptions) {
-        DateFormat dateFormat = DateFormat.getDateTimeInstance(DateFormat.DEFAULT, DateFormat.SHORT);
+    protected void setMetaData(StatusViewData.Concrete statusViewData, StatusDisplayOptions statusDisplayOptions, StatusActionListener listener) {
```

Comment:
Maybe add a comment that describes the expected format of the string, to save next reader from tracing through the code?

If I've read it correctly the format will be:

```
{visibilityIcon} {createdAt}{" • "?}{editedAtString?} • {appName?}
                                    -----------------   ---------
                                    Clickable           Clickable
```

where `?` suffix indicates optionality, and assuming that R.string.metadata_joiner is ` • ` (which it may not be).